### ISSUE TITLE
Compliance/Compliance Obligation - Generate Compliance Invoice

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-compliance#27]
#### Description:

As an external user, I can generate and view an invoice [operation's compliance summary page (compliance report 2)](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=32714-10350&t=tMX0hBZbzhSelgtv-4)

#### Acceptance Criteria:

Given I am on the compliance summary page
When I click the generate compliance invoice button, the invoice template generates

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=22760-18&t=sFE87VVDYnNuklhl-1)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #130 
- Blocking #137 
- Blocking #141

**Comment by NicoleGovvy** [comment]
@nanyangpro added an item about the payment instructions being on the invoice.

**Comment by nanyangpro** [comment]
Link to design is updated in the ticket description. Moving back to Backlog for future refinement and implementation.

**Comment by nanyangpro** [comment]
Moving back to Backlog with link to design updated in the ticket description and "Needs Wireframe" label removed (cc: @patrickisaac)

**Comment by patrickisaac** [comment]
Moved ticket to top of `Backlog` in the No Sprint section
cc: @vesselak

**Comment by patrickisaac** [comment]
Refinement Notes:
- Will be split to have PDF creation as a separate ticket

**Comment by nataliepiguliak** [comment]
@vesselak @fviduya per our morning discussion, I created a PR for fixing font and adjusting fields to be from Compliance Invoice, the rest of the items should be addressed after we receive data from the API
https://github.com/bcgov/cas-registration/pull/3232

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/ESB3dlFCIX5DjFmlzlI2tHYBTzRwGwYfsu9BLzRNhrVIWg?e=TdbqWV).

I can verify that:
- [x] Upon clicking `Generate Compliance Invoice`, the Compliance Invoice is generated and viewed in a new tab
- [x] The font used is correct
- [x] The contents are similar to the contents in the wireframe 

Moving to Pending PO for review. Thank you!

**Comment by vesselak** [comment]
Review the QA notes and the view of invoice in dev. Accepting ticket.
